libs/blocks/hero-marquee/hero-marquee.js

```diff
@@ -259,5 +259,14 @@ export default async function init(el) {
     }
   });
   decorateTextOverrides(el, ['-heading', '-body', '-detail'], mainCopy);
+
+  if (el.classList.contains('countdown-timer')) {
+    const { default: initCDT } = await import('../../features/cdt/cdt.js');
+    const classesToAdd = [];
+    if (el.classList.contains('center')) classesToAdd.push('center');
+    classesToAdd.push(el.classList.contains('dark') ? 'dark' : 'light');
+    await initCDT(copy, classesToAdd);
```

Comment:
i guess you are planning for different treatment depending on the the host block, but for now as it's not the case i would suggest to move that logic to initCDT
```suggestion
    await initCDT(copy, el.classList);
```